File: main/src/main/java/net/fwitz/math/main/escape/newton/NewtonsMethodJedSoftExample1.java
package net.fwitz.math.main.escape.newton;

import net.fwitz.math.main.complex.functions.examples.simple.JedSoftExample1;

public class NewtonsMethodJedSoftExample1 {
    public static void main(String[] args) {
        NewtonsMethodPlot.render(JedSoftExample1.FN_NAME + " (Newton's Method)", JedSoftExample1::fn);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/newton/NewtonsMethodJedSoftExample1.kt
package net.fwitz.math.main.binary.escape.newton

import net.fwitz.math.main.binary.complex.functions.examples.simple.JedSoftExample1

object NewtonsMethodJedSoftExample1 {
    @JvmStatic
    fun main(args: Array<String>) = NewtonsMethodPlot.render(
        name = JedSoftExample1.FN_NAME + " (Newton's Method)",
        f = JedSoftExample1::fn
    )
}